torch/_inductor/codegen/wrapper.py

```diff
@@ -1609,21 +1566,26 @@ def define_user_defined_triton_kernel(
         kwargs,
         restore_value_args,
         reset_to_zero_args,
+        grids: list[list[Union[int, sympy.Expr]]],
     ):
         from torch.utils._triton import patch_triton_dtype_repr
 
-        patch_triton_dtype_repr()
-
-        original_name = kernel.__name__
-
+        from ..runtime.triton_heuristics import (
+            config_to_dict,
+            FixedGrid,
+            PrecomputedGrid,
+        )
         from .common import (
             ConstexprArg,
             KernelArgType,
             SizeArg,
             TensorArg,
             TMADescriptorArg,
         )
+        from .triton import gen_common_triton_imports, TritonKernel
 
+        patch_triton_dtype_repr()
```

Comment:
I know this is existing code.. but the patch landed a while ago

Reply:
Let's remove this in a different PR in case this break something.